openwrt-openwisp-monitoring/files/monitoring.config

```diff
@@ -1,3 +1,4 @@
 config monitoring 'monitoring'
-	option monitored_interfaces '*'
+	option monitored_interfaces '"*"'
```

Comment:
requiring the user to remember to wrap the value around double quotes is really bad and weird. I think we can do better than this.